ra/registration-authority.go

```diff
@@ -161,6 +173,8 @@ func (ra *RegistrationAuthorityImpl) NewAuthorization(request core.Authorization
 		Status:         core.StatusPending,
 		Combinations:   combinations,
 		Challenges:     challenges,
+		// TODO(jsha): Pending authz should expire earlier than finalized authz.
```

Comment:
Actually, made it for you: https://github.com/letsencrypt/boulder/issues/782

Add comments if that's not accurate.